### DISCUSSION TITLE
Welcome to arozos Discussions!

### DISCUSSION THREAD

**Discussion by tobychui** [tobychui/arozos#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. You can also ask questions if you need help on anything.

**Comment by ghost** [comment]
Hi!

**Reply by tobychui** [reply]
Hi, you got any questions or ideas to share?

**Reply by ghost** [reply]
Hi tobychui.

> Hi, you got any questions or ideas to share?

thanks you comment here. 

So I liked this open project: arozos - I'll see how to help. I recently created a benchmarking of various operating systems.   *Would you be interested in seeing my open source project - that compares different operating systems or web operating systems?* (if you want i can leave a link here (I would be glad for some analysis) )

another question I have is... why arozos-os is not yet an organization in GitHub like Calamares, Ubuntu, Mint?

**Reply by tobychui** [reply]
Hi eantho,

You can feel free to add arozos in your list of comparison or leave links here (but no ads plz)

Regarding your second question, I do created an organization named aroz-online (Full name for ArozOS is Aroz Online System) over here [https://github.com/aroz-online](https://github.com/aroz-online) . However, as I am the only primary developer for this project, I am keeping it on my own account for easy management.

If you know Golang and want to become a primary contributor of this project, feel free to contact me via email and discuss the details. Thanks for you interest!

**Reply by ghost** [reply]
Hi tobychui.

> You can feel free to add arozos in your list of comparison or leave links here (but no ads plz)

thank you comment here again. Please, see this `https://github.com/eantho/delfos`

> Regarding your second question, I do created an organization named aroz-online (Full name for ArozOS is Aroz Online System) over here https://github.com/aroz-online . However, as I am the only primary developer for this project, I am keeping it on my own account for easy management.

thanks again.

> If you know Golang and want to become a primary contributor of this project, feel free to contact me via email and discuss the details. Thanks for you interest!

thank you for feedback - but I still don't have much interest in things like golang.
